portfolio/src/main/webapp/style.css

```diff
@@ -106,6 +106,24 @@ body, html {
   width: 800px;
 }
 
+.marker-input-box {
+  color: white;
+  background-color: #333;
+  border: 1px;
+  border-color: black;
+  border-style: solid;
+  float: left;
+  font-size: larger;
+  height: 50px;
+  margin-bottom:5px;
```

Comment:
Since all margin values are being set, consider using the shorthand:
`margin: 5px 5px 5xp 10px;`